fuzzers/Makefile

```diff
@@ -11,11 +13,36 @@ BITONLY=N
 all:
 
 define fuzzer
+
+# $(1) - Fuzzer name/directory
+# $(2) - Space seperated list of dependencies for the fuzzer.
+
+# Make the all target depend on the run.ok file for the fuzzer.
 all: $(1)/run.ok
+
+# Make the clean target run `make clean` in the fuzzer's directory.
 clean::
 	$$(MAKE) -C $(1) clean
+	rm -f $(1)/*.log
+
+# Describe how to create the fuzzer's run.ok file.
+# This command must start with a + to tell make to pass the jobserver
+# parameters downwards.
+ifeq ($(VERBOSE),Y)
+
+# When verbose we just call make directory
 $(1)/run.ok: $(addsuffix /run.ok,$(2))
-	$$(MAKE) -C $(1) run
+	$(MAKE) $(1)
```

Comment:
You do need double $$ here

Reply:
Fixed. (Also fixed the missing `-C $(1) run` bit).